inductiva/coastal/README.md

```diff
@@ -8,83 +8,61 @@ is performed using the [SWASH](https://swash.sourceforge.io/) simulator.
 ```python
 import inductiva
 
-bathymetry = inductiva.coastal.Bathymety.from_ascii_xyz_file(
-    ascii_xyz_file_path="590_HR_Lidar_Algarve.emo")
-```
-
-The bathymetry can be inspected by plotting it with a given resolution, in this
-case 200 m x 200 m:
-
-```python
-bathymetry.plot(x_resolution=200, y_resolution=200)
+bathymetry = inductiva.coastal.Bathymetry.from_random_depths(
+        x_range=(0, 100),
+        y_range=(0, 100),
+        x_num=100,
+        y_num=100,
+        max_depth=50)
 ```
 
-![Raw bathymetry.](resources/media/bathymetry.png)
-
-Raw bathymetry data will often span an area too large and sparsely sampled to
-be used in a simulation. In this case, the user can crop the bathymetry to a
-smaller area of interest:
+This bathymetry is defined in a 50x50 grid, that represent a domain 100 m
+wide and long. It can be inspected by plotting it.
 
 ```python
-bathymetry = bathymetry.crop(x_range=(51000, 52000), y_range=(12150, 13000))
 bathymetry.plot()
 ```
 
-![Cropped bathymetry.](resources/media/bathymetry_cropped.png)
+![Raw bathymetry.](/resources/bathymetry/bathymetry_random.png)
 
-To be used in a simulation, the bathymetry data must be interpolated to a
-regular grid with custom resolution, in this case 5 m x 5 m. In case some
-positions are missing in the bathymetry data (see e.g. the lower left corner in
-the figure above), the user can choose to fill them with a constant depth or the
-nearest depth value:
-
-```python
-bathymetry = bathymetry.to_uniform_grid(x_resolution=5,
-                                        y_resolution=5,
-                                        fill_value="nearest")
-```
-
-When interpolated to a regular grid, the bathymetry is ready to be used in a
-simulation scenario. In this case, we set the wave source location to the south
-boundary of the domain (i.e. the lower y boundary), with a wave amplitude of 5 m
-and a wave period of 5.5 s:
+The bathymetry is ready to be used in a simulation scenario. In this case, we set
+the wave source location to the west boundary of the domain (i.e. the lower x
+boundary), with a wave amplitude of 5 m and a wave period of 5.5 s:
 
 ```python
 scenario = inductiva.coastal.CoastalArea(bathymetry=bathymetry,
-                                         wave_source_location="S",
+                                         wave_source_location="W",
                                          wave_amplitude=5,
                                          wave_period=5.5)
 ```
 
-In case the bathymetry is not defined on a regular grid when creating a
-scenario, interpolation to a regular grid is automatically attempted with a
-default resolution of 2 m x 2 m and no pre-configured strategy to fill depths at
-positions missing in the bathymetry.
-
 Once the scenario is created, run the simulation as follows:
 
 ```python
-task = scenario.simulate(simulation_time=100, output_time_step=1)
+task = scenario.simulate(simulation_time=30, output_time_step=1)
```

Comment:
I am nor sure if I understand what is the output_time_step. Are we dumping frames every second? Can you add a comment on that?